packages/grid/src/GridRenderer.ts

```diff
@@ -2335,6 +2335,8 @@ export class GridRenderer {
       visibleColumns.push(i);
     }
 
+    // The headers expect the context to be at the edge...
+    context.translate(-gridX, 0);
```

Comment:
The translate before this is `draggingLeft - originalLeft`. Is this the same as `-gridX` in all cases? Also clarify what "the edge" is. Edge of the canvas? Edge of some bounding box?